packages/ERTP/src/amountMath.chainmail

```diff
@@ -25,98 +26,92 @@ struct Extent {
 }
 
 /**
- * Operations on Units.
+ * Logic for manipulating amounts.
  *
- * Units are the canonical description of tradable goods. They are manipulated
- * by mints, and represent the goods and currency carried by purses and
+ * Amounts are the canonical description of tradable goods. They are manipulated
+ * by issuers and mints, and represent the goods and currency carried by purses and
  * payments. They can be used to represent things like currency, stock, and the
  * abstract right to participate in a particular exchange.
  *
  */
-interface UnitOps (Units (Brand, Extent)) {
+interface AmountMath (Amount (Brand, Extent)) {
 
-  /** Return this unitOps's brand. */
+  /** Return the brand. */
   getBrand() -> (Brand);
 
-  /** Get the name of the extentOps used. */
-  getExtentOpsName -> (String);
+  /** Get the name of the mathHelpers used. */
+  getMathHelpersName -> (String);
 
-  /**  Make units from an extent by adding the brand. */
-  make(allegedExtent :Extent) -> (Units);
+  /**  Make an amount from an extent by adding the brand. */
+  make(allegedExtent :Extent) -> (Amount);
 
   /**
-   * Make sure these units (or extent) are valid and return them if so.
+   * Make sure this amount (or extent) is valid and return it if so.
    */
-  coerce(allegedUnitsOrExtent :Units) -> (Units);
+  coerce(allegedAmountOrExtent :Amount) -> (Amount);
 
   /** Remove the brand and return the extent. */
-  extent(units :Units) -> (Extent);
+  extent(amount :Amount) -> (Extent);
 
-  /** Return the units representing an empty amount. This is the
-   * identity element in ExtentOps.
+  /** Return the amount representing an empty amount. This is the
+   * identity element in MathHelpers.
    */
-  empty() -> (Units);
+  getEmpty() -> (Amount);
 
-  /** Return true if the Units is empty. Otherwise false. */
-  isEmpty(units :Units) -> (boolean);
+  /** Return true if the Amount is empty. Otherwise false. */
+  isEmpty(amount :Amount) -> (boolean);
 
-  /** Returns true if the leftUnits contains the rightUnits. */
-  includes(leftUnits :Units, rightUnits :Units) -> (boolean);
+  /** Returns true if the leftAmount is greater than or equal to the rightAmount. */
```

Comment:
This is why I'm suspicious of renaming it isGTE. Come operator overloading, we'll probably use `>=` so we should keep `isGTE`. But we need to explain the inclusion relationship of the implied rights. Is rectangle A >= rectangle B? Well, it depends on how we generalize `>=` to non-scalars, which isn't obvious. Does rectangle A include rectangle B? Ah, now the reader knows what question is being asked.

Reply:
Yup, this needs more explanation